packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -859,7 +859,7 @@ const defaultConfigs = {
     type: "expression",
     expression: `
         ETHBTC_FV = ETH\\/BTC * PERP_FRM;
-        round(max(-0.00001, min(0.00001, (ETHBTC_FV - ETHBTC_PERP) / ETHBTC_FV / 86400)), 9)
```

Comment:
Removing this `round` fixes the GCP issue.

Reply:
This is so wack. I dont understand how this could cause GCP to blow out like this. 

One explanation could be that when you've been testing this locally you are running it on your Mac within your dev environment. GCP is running it within a docker container. it is possible that there is some execution difference between the two that breaks this js math logic?

Reply:
I've been running the spoke on GCP actually, and I can replicate the GCP logging bug unless I remove this `round`